src/harness/certs/README.md

```diff
@@ -12,24 +12,32 @@ WINNF-15-S-0065" document. Naming and base configuration are issued from
 https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/cert
 
 ```
-                  -------------root_ca --------------------------------------                
-                 /                   \         				     \                 
-                /                     \         			      \            
-          sas_ca--------              cbsd_ca   			   proxy_ca
-          /    \        \                \              	  		\
-         /      \        \                \              	  		 \
-   admin_client  server  sas     client|device_[a|c]|corrupted_client        domain_proxy
-		          |       wrong_type_client|client_expired           corrupted_domain_proxy
-                   corrupted_sas  client_inapplicable                        wrong_type_domain_proxy
-                   sas_expired                                               domain_proxy_expired
-                                                                             domain_proxy_inapplicable
-
- 
-unrecognized_ca             non_cbrs_root_ca----------------------------------------------
-         |                     |                                 \	                  \
-unrecognized_device|        non_cbrs_root_signed_cbsd_ca   non_cbrs_root_signed_sas_ca  non_cbrs_root_signed_oper_ca
-unrecognized_sas               |                                  |                        |
-unrecognized_domain_proxy   non_cbrs_signed_device         non_cbrs_root_signed_sas     non_cbrs_signed_domain_proxy
+             ------------------root_ca ------------------                
```

Comment:
Please remove the tab characters and trailing spaces from this diagram.

Reply:
The   trailing spaces has been removed now

Reply:
Actual:
```
             ------------------root_ca ------------------
            /                     |         <TAB>         \<SPACE>
           /                      |          <TAB>          \<SPACE>
       sas_ca--------          cbsd_ca   <TAB>        proxy_ca
       /    \        \            |<TAB><TAB><TAB>    \
      /      \        \           |                   <TAB>     \
admin_client sas    server     client                   domain_proxy
              |                corrupted_client         corrupted_domain_proxy
 <TAB>  corrupted_sas        wrong_type_client        wrong_type_domain_proxy
          sas_expired          client_expired           domain_proxy_expired
          blacklisted_sas      client_inapplicable      domain_proxy_inapplicable<SPACE>
                               blacklisted_client       blacklisted_domain_proxy<SPACE>
<SPACE>
<SPACE>
            ------------------non_cbrs_root_ca----------------------
           /                          |        <TAB>                    \
non_cbrs_root_signed_cbsd_ca   non_cbrs_root_signed_sas_ca  non_cbrs_root_signed_oper_ca
          |                           |                              |
non_cbrs_signed_device         non_cbrs_root_signed_sas     non_cbrs_signed_domain_proxy


                             unrecognized_ca<SPACE>
                                   |<SPACE>
                          unrecognized_device<SPACE>
                          unrecognized_sas<SPACE>
                          unrecognized_domain_proxy<SPACE>
```

Expected:
```
             ------------------root_ca ------------------
            /                     |                      \
           /                      |                       \
       sas_ca--------          cbsd_ca                  proxy_ca
       /    \        \            |                         \
      /      \        \           |                          \
admin_client sas    server     client                   domain_proxy
              |                corrupted_client         corrupted_domain_proxy
          corrupted_sas        wrong_type_client        wrong_type_domain_proxy
          sas_expired          client_expired           domain_proxy_expired
          blacklisted_sas      client_inapplicable      domain_proxy_inapplicable
                               blacklisted_client       blacklisted_domain_proxy


            ------------------non_cbrs_root_ca----------------------
           /                          |                             \
non_cbrs_root_signed_cbsd_ca   non_cbrs_root_signed_sas_ca  non_cbrs_root_signed_oper_ca
          |                           |                              |
non_cbrs_signed_device         non_cbrs_root_signed_sas     non_cbrs_signed_domain_proxy


                             unrecognized_ca
                                   |
                          unrecognized_device
                          unrecognized_sas
                          unrecognized_domain_proxy
```

Reply:
Thank you! Fixed this with your expected output.